src/components/ProjectCard.tsx

```diff
@@ -33,7 +33,7 @@ const ProjectCard = ({
             />
           </ImageContainer>
         ) : (
-          <Placeholder
+          <StyledPlaceholder
```

Comment:
If this placeholder component is used in several places in project, It is good practice to  have one component and reuse that. (I see you have same in at least one place -in ProjectCard.tsx)